include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1813,6 +1813,11 @@ class FEEvaluationAccess<1, 1, Number, is_face, VectorizedArrayType>
   gradient_type
   get_gradient(const unsigned int q_point) const;
 
+  /** @copydoc FEEvaluationBase<1,1,Number,is_face>::get_divergence()
```

Comment:
```suggestion
  /** 
   * @copydoc FEEvaluationBase<1,1,Number,is_face>::get_divergence()
```

Reply:
Do you think it is really necessary to change this? I used the same formatting as other functions do, see e.g.

https://github.com/dealii/dealii/blob/1ec834c778615948de2fb6a33597e3c55dc02ce4/include/deal.II/matrix_free/fe_evaluation.h#L1487-L1505

Reply:
Yes ;) Could you also fix the formatting for the methods below and above (its just 34 instances)?

Reply:
Done ;-).